tests/029.solrdocument_serialize_php81.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+SolrDocument - clone
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 80100) die("skip PHP 8.1+ only"); ?>
+--FILE--
+<?php
+
+require_once "bootstrap.inc";
+
+$fixtureXml = file_get_contents(EXAMPLE_RESPONSE_XML_2);
+
+$response = SolrUtils::digestXmlResponse($fixtureXml, SolrResponse::PARSE_SOLR_DOC);
+
+$doc =$response->response->docs[1];
+
+$serializedString = serialize($doc);
+print_r($serializedString);
+echo PHP_EOL;
+print_r(unserialize($serializedString)->toArray());
+?>
+--EXPECT--
+O:12:"SolrDocument":1:{s:3:"xml";s:172:"<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Generally, avoid hard-coded string lengths in tests if there are line-breaks; this might not work on Windows (LF vs. CRLF). Instead you can use a placeholder:
```suggestion
O:12:"SolrDocument":1:{s:3:"xml";s:%d:"<?xml version="1.0" encoding="UTF-8"?>
```

Reply:
I agree, but I don't want to change these tests, they are old and tried (I hope).